pkg/handlers/webhookreceiver.go

```diff
@@ -390,7 +397,11 @@ func (h *WebhookReceiverHandler) updateNotificationStatus(n *oav1alpha1.Notifica
 					// Update the timestamp for the ServiceLogSent
 					_ = status.SetStatus(oav1alpha1.ConditionAlertFiring, "Alert is not firing", corev1.ConditionFalse, timeNow)
 					_ = status.SetStatus(oav1alpha1.ConditionAlertResolved, "Alert resolved", corev1.ConditionTrue, timeNow)
-					_ = status.SetStatus(oav1alpha1.ConditionServiceLogSent, "Service log sent for alert resolved", corev1.ConditionTrue, timeNow)
+					if len(n.ResolvedDesc) > 0 {
+						_ = status.SetStatus(oav1alpha1.ConditionServiceLogSent, "Service log sent for alert resolved", corev1.ConditionTrue, timeNow)
+					} else {
+						status.ServiceLogSentCount--
```

Comment:
Why do we need to decrement this?

Reply:
> Why do we need to decrement this?

This is for the total `serviceLogSentCount`. I found this issue when I was doing the test. Can refer to the logic [here](https://github.com/openshift/ocm-agent-operator/blob/master/api/v1alpha1/managednotification_types.go#L266). After the [update status](https://github.com/openshift/ocm-agent/blob/master/pkg/handlers/webhookreceiver.go#L406), the count for sending the service log will be increased which should be wrong since we haven't sent it for resolved ones. I do not want to change more for OAO code, so figured out the idea here.